javascript/javascript1/week1/name4agame.js

```diff
@@ -0,0 +1,7 @@
+//Ez Name 4 a game (Startup name generator) 
+
+const firstWords=["Incredible","Easy","Beautiful","Happy","Angry","Dumb","Crazy","Wise","Magic","Amazing"]
+const seconedWords=["Fox","Dog","Cat","Horse","Octopus","Bird","Llama","Whale","Crocodile","Dinosaur"]
+const randomNumber = Math.floor(Math.random() * 7);
```

Comment:
it would be better to use 
```Math.floor(Math.random() *  firstWords.length)``` 
With this approach there won't be need to change constant 7 when words are added/removed